source/ore/security.rst

```diff
@@ -0,0 +1,5 @@
+========
+Security
```

Comment:
This file is currently not linked anywhere. It should probably be removed or linked, so that its obvious that it is missing.